### ISSUE TITLE
Error running 0.11.0

### ISSUE THREAD

**Issue by rclmenezes** [cloudposse/terraform-aws-ecs-codepipeline#15]
When I run:
```
module "ecs_web_app" {
  source                       = "git::https://github.com/cloudposse/terraform-aws-ecs-web-app.git?ref=tags/0.10.0"
  name                         = "web"
  namespace                    = "upright"
  stage                        = "prod"
  vpc_id                       = "${module.vpc.vpc_id}"
  listener_arns                = ["${module.alb.listener_arns}"]
  listener_arns_count          = "1"
  aws_logs_region              = "us-east-1"
  ecs_cluster_arn              = "${aws_ecs_cluster.default.arn}"
  ecs_cluster_name             = "${aws_ecs_cluster.default.name}"
  ecs_private_subnet_ids       = ["${values(module.private_subnets.az_subnet_ids)}"]
  alb_ingress_healthcheck_path = "/healthz"
  alb_ingress_paths            = ["/*"]
  codepipeline_enabled         = "false"

  github_oauth_token           = "XXX"
  repo_name = "seedfi"
  repo_owner = "rclmenezes"
}
```

I get:

```
Error: Error running plan: 1 error(s) occurred:

* module.ecs_web_app.module.ecs_codepipeline.aws_codepipeline_webhook.webhook: 1 error(s) occurred:

* module.ecs_web_app.module.ecs_codepipeline.aws_codepipeline_webhook.webhook: Resource 'aws_codepipeline.source_build_deploy' not found for variable 'aws_codepipeline.source_build_deploy.name'
```

Works fine on 0.10.0

**Comment by rclmenezes** [comment]
I *think* this is related to `terraform-aws-ecs-codepipeline`, but it could be a `terraform-aws-ecs-web-app` issue as well

**Comment by osterman** [comment]
@rclmenezes can you try setting `webhook_enabled=false`?

**Comment by osterman** [comment]
@rclmenezes I am getting this now too.

 > `Resource 'aws_codepipeline.source_build_deploy' not found for variable 'aws_codepipeline.source_build_deploy.name'`

Looking into it.

**Comment by osterman** [comment]
Fixed in `0.4.1`

**Comment by rclmenezes** [comment]
Thanks!
